src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/attribute_editor.cljs

```diff
@@ -66,43 +125,52 @@
                           :content-renderer
                           (fn [index]
                             (icons/icon {:style {:color (:exception-state style/colors)
-                                                 :verticalAlign "middle"
+                                                 :verticalAlign "middle" :height 26
                                                  :cursor "pointer"}
                                          :onClick #(swap! state update :attributes utils/delete index)}
                                         :delete))}
                          {:header "Key" :starting-width 300 :as-text (constantly nil)
```

Comment:
UX tweak: should the starting val be bigger, to line up to where the values are for dataset attributes?